### DISCUSSION TITLE
Loopback Connector for AWS Neptune with Gremlin

### DISCUSSION THREAD

**Discussion by wesleymilan** [loopbackio/loopback-next#7762] [Ideas]
Hello everyone,
I've just created a new connector for AWS Neptune using Gremlin Query Language, is there anyone interested in contributing to my project?
Need ninjas to help to implement new tests and validating complex queries.

https://www.npmjs.com/package/loopback-connector-neptune

Thank you

**Comment by achrinza** [comment]
Looks great, @wesleymilan!

Would you like to add it to the [Community Extensions](https://github.com/loopbackio/loopback-next/blob/71913afd6b6246ae79218e239d2de84f6a36552e/docs/site/Community-extensions.md) docs?

**Reply by wesleymilan** [reply]
That would be great.
I'm starting a social media project where I'll test and improve this connector under fire, as soon as it's fully tested and stable I'd appreciate seeing it on the official community connectors list.

Thank you very much @achrinza
